src/expr/dtype.cpp

```diff
@@ -260,7 +260,7 @@ void DType::setSygus(TypeNode st, Node bvl, bool allowConst, bool allowAll)
   // Notice we only want to do this for sygus datatypes that are user-provided.
   // At the moment, the condition !allow_all implies the grammar is
   // user-provided and hence may require a default constant.
-  // TODO (https://github.com/CVC4/cvc4-projects/issues/38):
+  // TODO (https://github.com/cvc5/cvc5-projects/issues/38):
```

Comment:
This issue has been closed...

Reply:
Yeah, I noticed that but from a quick check, it looked like the comment is still relevant. I've already pinged @ajreynol about it. We'll see what he says :)

Reply:
I have discussed it with Andy and removed the reference to the issue. The comment about the alternative design is not important enough to warrant an issue.